multitail: 7.0.0 -> 7.1.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/project/multitail/versions.


meta.description for multitail is: tail on Steroids

meta.homepage for multitail is: https://github.com/halturin/multitail




###### Updates performed
- Version update

###### To inspect upstream changes



- [Compare changes on GitHub](https://github.com/folkertvanheusden/multitail/compare/7.0.0...7.1.1)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 7.1.1 with grep in /nix/store/7z2wv2jv965vyx34517brdhhl2a78c8j-multitail-7.1.1
- found 7.1.1 in filename of file in /nix/store/7z2wv2jv965vyx34517brdhhl2a78c8j-multitail-7.1.1

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
multitail
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/7z2wv2jv965vyx34517brdhhl2a78c8j-multitail-7.1.1 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A multitail https://github.com/r-ryantm/nixpkgs/archive/c4e8ee187b6c146bc125eb99f73ef604f4e1bb4c.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/c4e8ee187b6c146bc125eb99f73ef604f4e1bb4c#multitail
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/7z2wv2jv965vyx34517brdhhl2a78c8j-multitail-7.1.1
ls -la /nix/store/7z2wv2jv965vyx34517brdhhl2a78c8j-multitail-7.1.1/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>multitail</li>
  </ul>
</details>

---

###### Maintainer pings

cc @matthiasbeyer for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
multitail: 7.0.0 -> 7.1.1

MinerSebas reviewed (commented):
The value `https://github.com/halturin/multitail` of `meta.homepage` is out of date, as the fetcher actually sources it from `https://github.com/folkertvanheusden/multitail/` instead.

Looking further the `folkertvanheusden` version is also Apache2, compared to the `halturin` version which is GPL2.

natsukium committed:
multitail: update meta

- use the fork url as the homepage instead of the original one
- change the license to asl20

matthiasbeyer commented:
How can one version be apache and one GPL?
That sounds a lot like different tools to me, tbh.
